lib/opentelemetry-node.js

```diff
@@ -122,7 +142,11 @@ function createSpan (tracer, msg, nodeDefinition, node, isNotTraced) {
     } else {
       if (nodeDefinition.type === 'http in') {
         // try to get trace context in incoming http request headers
-        ctx = propagation.extract(context.active(), msg.req.headers)
+        ctx = propagator.extract(context.active(), msg.req.headers, defaultTextMapGetter)
```

Comment:
I don't think it's necessary to provide the default TextMapGetter (`defaultTextMapGetter`).
[Docs](https://open-telemetry.github.io/opentelemetry-js/interfaces/_opentelemetry_api.TextMapPropagator.html#extract) says _"If undefined, keys will be all own properties, and keys will be accessed by direct object access."_.
Can you test without this?
If ok, remove also line 16.

Reply:
:/ ok, I checked now, only in the propagation export of otel/api the default is set, the moment a custom propagator is built the getter is not set.

It would have been nice if the CompositePropagator would also set a default like the TextMapPropagator

Reply:
Ok, thanks for this analyse.